igvm/cli.py

```diff
@@ -198,6 +198,24 @@ def parse_args():
         action='store_true',
         help='Perform IP address change offline',
     )
+    subparser.add_argument(
```

Comment:
Why adding migration support there is a dedicated command igvm migrate. I don't why this is beneficial could you please explain.

Reply:
Because some users demanded functionality of changing IP address of a VM to a subnet not supported on original HV. To implemment it migration is necessary. Other way of doing it would be to have `igvm migrate --change-address` but then there would be two ways of changing address. I would personally be totally fine with *not* providing users such functionality but apparently it's a hot topic and probably the most wanted one.